style..css

```diff
@@ -534,6 +535,602 @@ img {
+            display: flex;
+            flex-direction: column; 
+            gap: 10px;
+            width: 100%;
+            overflow-x: hidden; 
+        }
+        .card {
+            flex: 1 1 100%; 
+            box-sizing: border-box;
+            margin-bottom: 20px; 
+            border: 1px solid #ddd; 
+            overflow: hidden; 
+            background-color: #fff;  
+            padding-top: 20px;
+            width: 100%;
+            overflow-x: hidden;
+        }
+        .header-container {
+            flex-direction: column;
+            text-align: left;
+            width: 100%;
+            overflow-x: hidden;
+        }
+        .click {
+            margin-top: -350px;
+            margin-left: -200px;
+        }
+        .card img {
+            width: 100%; 
+            height: auto;
+        }
+        .card-content {
+            padding: 10px;
+        }
+        .card-content h4 {
+            margin: 0 0 10px; 
+            font-size: 0.4em; 
+            color: #333;
+            text-align: left;
+        }
+        .location {
+            display: flex;
+            align-items: center;
+            gap: 5px; 
+        }
+        .location img {
+            width: 14px; 
+            height: auto;
+        }
+        .location h6 {
+            margin: 0;
+            font-size: 0.2em; 
+            color: #666; 
+        }
+        section h1 {
+            font-size: 0.9em;
+            margin-bottom: 10px;
+            text-align: left;
+            width: 100%;
+        }
+        section h2 {
+            font-size: 0.4em;
+            color: #6c757d;
+            text-align: left; 
+            width: 100%;
+        }
+        .inner-card-container {
+            display: flex; 
+            flex-direction: column; 
+            gap: 20px; 
+            justify-content: center;
+            width: 100%;
+            height: auto; 
+            overflow-x: hidden;
+        }
+        .best {
+            margin-top: -100px;
+        }
+        .guide {
+            margin-top: -100px;
+        }
+        .always {
+            margin-top: -100px;
+        }
+        .text-content {
+            width: 100%;
+        }
+        .trip {
+            flex-direction: column;
+        }
+        .p-travel {
+            font-size: 0.9em;
+        }
+        .container {
+            flex-direction: column;
+            overflow-x: hidden;
+        }
+        .header-container {
+            text-align: left;
+            width: 100%;
+            overflow-x: hidden;
+        }
+        .content {
+            flex-direction: column; 
+            align-items: flex-start;
+            padding-left: 0;
+            padding-right: 0;
+        }
+        .text-group {
+            font-size: 1.3em;
+            padding-left: 0;
+            padding-right: 0;
+        }
+        .group {
+            margin-bottom: 20px; 
+        }
+        .content-pic {
+            width: 100%;
+        }
+    }
+    
+}
```

Comment:
Hi @fatimaabubakaradam ,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    <img src="https://i.imgur.com/nvrw8p7.gif" alt="Remote Collaboration" style="width:100%;">
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._